examples/player/simid_player.html

```diff
@@ -4,7 +4,7 @@
   <script src="../simid_protocol.js"></script>
   <script src="simid_player.js"></script>
   <style>
-    .simid_creative {
```

Comment:
what do you think of just removing the simid_player changes from this PR entirely, since you might conflict with kaleabs changes.